coffeecard/CoffeeCard.Library/Migrations/20230711144034_AddPurchaseTypes.cs

```diff
@@ -0,0 +1,97 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace CoffeeCard.Library.Migrations
+{
+    public partial class AddPurchaseTypes : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<int>(
+                name: "PurchaseId",
+                schema: "dbo",
+                table: "Vouchers",
+                type: "int",
+                nullable: true);
+
+            migrationBuilder.AddColumn<string>(
+                name: "Type",
+                schema: "dbo",
+                table: "Purchases",
+                type: "nvarchar(max)",
+                nullable: false,
```

Comment:
We probably need to do this in two steps; one where Type is nullable and allows us to populate the column. A second where we set it to non nullable